.github/workflows/stackhpc-container-image-build.yml

```diff
@@ -207,6 +215,109 @@ jobs:
           path: ${{ matrix.distro }}-container-images
           retention-days: 7
 
+      - name: Generate list of images to scan/push
+        run: |
+          # Clean up any stale data
+          rm -rf ${{ matrix.distro }}-docker-images.txt
+
+          # Make a file of imagename:tag
+          grep --invert-match --no-filename ^REPOSITORY ${{ matrix.distro }}-container-images |\
+          sed 's/ \+/:/g' |\
+          cut -f 1,2 -d: > ${{ matrix.distro }}-docker-images.txt
+        
+      - name: Scan built container images
+        run: |
+          set -euo pipefail
```

Comment:
I guess it would be nice if this was in a script in `tools/` that could be run manually.

Reply:
done